### PR TITLE
Functionality to use URL

### PR DESCRIPTION
**Pull request by yrki**
In my use-case it is useful to get the data from an url instead of a local path.

### CODE REVIEW

**Comment by yrki** [comment]
@christianhelle I haven't tested it much yet. Just a concept.

**Comment by christianhelle** [comment]
@yrki first of all, thank you for this contribution.

I like the idea and I can see how many others will benefit from this

**Comment by christianhelle** [comment]
The smoke tests all pass which should verify that existing features didn't break

File: src/Refitter.Tests/PathParametersWithUrlTests.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+using FluentAssertions.Execution;
+using Refitter.Core;
+using Refitter.Tests.Build;
+using Xunit;
+
+namespace Refitter.Tests;
+
+public class PathParametersWithUrlTests
+{
+    private const string OpenApiSpec = @"
```
Comment (christianhelle) [inline src/Refitter.Tests/PathParametersWithUrlTests.cs:11]: This const doesn't seem to be used, but I can just remove that myself once this gets merged in

**Review by christianhelle — APPROVED** [review]
All in all, thislooks good to me

**Comment by christianhelle** [comment]
@yrki This feature is now [released](https://github.com/christianhelle/refitter/releases/tag/0.4.1) and [published to nuget.org](https://www.nuget.org/packages/Refitter/0.4.1)

Thanks again!

**Comment by yrki** [comment]
Great! Thanks! :)

**Pull request merged**